File: content/guides/source/includes/code/python/crud_read_final_less_than.py
from pymongo import MongoClient

# Replace the uri string with your MongoDB deployment's connection string.
uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority"

client = MongoClient(uri)
coll = client.sample_guides.planets

# find code goes here
cursor = coll.find({"surfaceTemperatureC.mean": {"$lt": 15}})

for doc in cursor:
    print(doc)

# Close the connection to MongoDB when you're done.
client.close()

File: content/guides/source/includes/code/node/crud-read-final-less-than.js
const { MongoClient } = require("mongodb");
// Replace the uri string with your MongoDB deployment's connection string.
const uri =
  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority";
const client = new MongoClient(uri);
async function run() {
  try {
    await client.connect();
    const coll = client.db("sample_guides").collection("planets");

    // find code goes here
    const cursor = coll.find({ "surfaceTemperatureC.mean": { $lt: 15 } });

    await cursor.forEach(console.log);
  } finally {
    // Ensures that the client will close when you finish/error
    await client.close();
  }
}
run().catch(console.dir);
